README.md

```diff
@@ -1,69 +1,78 @@
 [![Build Status](https://travis-ci.org/mentorpaired/core.svg?branch=staging)](https://travis-ci.org/mentorpaired/core) [![Coverage Status](https://coveralls.io/repos/github/mentorpaired/core/badge.svg?branch=staging)](https://coveralls.io/github/mentorpaired/core?branch=staging)
 
 # CORE
+
 Core is the backend of MentorPaired, an app which connects software engineering mentors to mentees who seek to learn any programming language.
 
 #### Tech
+
 Core is written in Python3 and Django 3.0.3.
```

Comment:
I see that we currently have `Django 3.0.7.` in the migration files and requirements.txt, so this should be updated on the readme as well